.github/workflows/reusable/build-and-test.yml

```
@@ -0,0 +1,105 @@
+      backend-image-tag: ${{ steps.tags.outputs.backend-tag }}
+      frontend-image-tag: ${{ steps.tags.outputs.frontend-tag }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
+
+      - name: Set image tags
+        id: tags
+        run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2086:info:1:55: Double quote to prevent globbing and word splitting

(shellcheck)